test/parser/controlFlow/__snapshots__/While.test.js.snap

```diff
@@ -3,7 +3,9 @@
 exports[`parser while statements while with exit 1`] = `
```

Comment:
If I'm honest, I'm scrolling past all of these snapshots on the assumption you ran `yarn test -u` to generate the new versions and didn't hand-edit them.

…I _really_ hope you didn't hand-edit them, because that's a pain!

Reply:
Correct! `yarn test -u` did the trick. Thanks for the tip!